src/content/development/building-testing/ci-maintenance/_index.en.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "CI/CD Maintenance"
+date: 2022-01-05T16:03:26+01:00
+weight: 10
+---
+
+This page documents the maintenance of Submariner's CI/CD for developers.
+
+## Kubernetes Versions
+
+The versions of Kubernetes tested in Submariner's CI need to be updated for new [Kubernetes releases](https://kubernetes.io/releases/).
+
+Submariner's policy is to support all versions upstream-Kubernetes supports and no EOL versions.
+
+The versions that should be used in CI are described below.
+
+<!-- markdownlint-disable line-length -->
+CI | Kubernetes Version | Notes
+:--- | :---- | :----
+Most CI | Latest | CI should run against the latest Kubernetes version by default.
+Basic Kubernetes Support | All non-latest supported versions | One defaults-only E2E for each non-latest supported Kubernetes version.
+Full Kubernetes Support | All non-latest supported versions | (*Doesn't exist yet*) Workflow to run all CI for all of the currently-supported, non-latest Kubernetes versions. Run periodically, on releases, or manually by adding a label.
```

Comment:
```suggestion
Full Kubernetes Support | All non-latest supported versions | Workflow to run all CI for all of the currently-supported, non-latest Kubernetes versions. Run periodically, on releases, or manually by adding a label.
```

This does exist now, everything is done for this task except waiting on the re-review.

https://github.com/submariner-io/submariner-website/issues/624#issuecomment-992808761